ur_robot_driver/src/dashboard_client_ros.cpp

```diff
@@ -120,6 +120,10 @@ DashboardClientROS::DashboardClientROS(const ros::NodeHandle& nh, const std::str
   // Query whether the current program is saved
   is_program_saved_service_ = nh_.advertiseService("program_saved", &DashboardClientROS::handleSavedQuery, this);
 
+  // Query whether the robot is in remote control
+  is_in_remote_control_service_ =
+      nh_.advertiseService("in_remote_control", &DashboardClientROS::handleRemoteControlQuery, this);
```

Comment:
```suggestion
      nh_.advertiseService("is_in_remote_control", &DashboardClientROS::handleRemoteControlQuery, this);
```

I'd prefer if the service names reflect the dashboard commands as close as possible. I know that we haven't been on spot there in the past and also the commands themselves are not consistent in terms of syntax, but it would be great to have a look on this in the future.